krun.py

```diff
@@ -206,36 +190,29 @@ def main(parser):
     mailer = Mailer(mail_recipients, max_mails=config.MAX_MAILS)
 
     try:
-        inner_main(mailer, config, args)
+        inner_main(mailer, on_first_invokation, config, args)
     except util.FatalKrunError as e:
         subject = "Fatal Krun Exception"
         mailer.send(subject, e.args[0], bypass_limiter=True)
         util.run_shell_cmd_list(config.POST_EXECUTION_CMDS)
```

Comment:
I don't think we should leave the reordering though -- it's a one line fix, and we'll forget it otherwise.